### ISSUE TITLE
[BUG] bug with relaunch on flyteadmin >=0.4.7

### ISSUE THREAD

**Issue by jganbat-fn** [flyteorg/flyte#900]
**Describe the bug**
I am on a remote sandbox on GCS and when attempting to relaunch from the Flyte UI causes 500 error.
Flyte versions: 
- `flyteadmin:v0.4.9`
- `flyteconsole:v0.19.6`
- `flytepropeller:v0.10.1`
- `datacatalog:v0.3.2`

Current behavior
Relaunch request is
`Request URL: http://localhost:51234/console/cors_proxy/gs://freenome-user-data-jganbat/metadata/flyte-sandbox-jganbat/main/t5rr3a1ctl/inputs`
Flyteconsole error is
```
::ffff:10.0.164.168 - - [08/Apr/2021:02:29:03 +0000] "GET /console/cors_proxy/gs://freenome-user-data-jganbat/metadata/flyte-sandbox-jganbat/main/t5rr3a1ctl/inputs HTTP/1.1" 500 148 "http://localhost:51234/console/projects/flyte-sandbox-jganbat/domains/main/executions/t5rr3a1ctl" "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36"
TypeError [ERR_INVALID_PROTOCOL]: Protocol "gs:" not supported. Expected "http:"
at new ClientRequest (_http_client.js:155:11)
at Object.request (http.js:50:10)
at /app/corsProxy.js:27:34
at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
at trim_prefix (/app/node_modules/express/lib/router/index.js:317:13)
at /app/node_modules/express/lib/router/index.js:284:7
at Function.process_params (/app/node_modules/express/lib/router/index.js:335:12)
at next (/app/node_modules/express/lib/router/index.js:275:10)
at jsonParser (/app/node_modules/body-parser/lib/types/json.js:119:7)
at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
TypeError [ERR_INVALID_PROTOCOL]: Protocol "gs:" not supported. Expected "http:"
```

**Expected behavior**
Successful relaunch of the workflow

**[Optional] Additional context**
***To Reproduce***
1. have a remote sandbox
2. on `>=flyteadmin:v0.4.7`
3. Run example workflow
4. From the Flyte UI attempt to relaunch the workflow 
***Screenshots***
![image](https://user-images.githubusercontent.com/67763150/113962292-3f910500-97dc-11eb-9cd6-550dff2a24cb.png)

**Comment by EngHabu** [comment]
Looks like it's a bug in the pricy. As a workaround, are you able to serve console and admin from the same port? It looks like you're portforwarding? Maybe portforward envoy service instead?

**Comment by katrogan** [comment]
given that flyteadmin already has the primary workflow execution inputs recorded in the db we shouldn't even need console (or any caller) to pass inputs in the relaunch request

**Comment by jganbat-fn** [comment]
Hi @katrogan is it possible to share the slack conversation; looks like solution to this issue has been found there :D.

**Comment by kumare3** [comment]
@jsonporter this also has a corresponding UI Fix

**Comment by EngHabu** [comment]
Should be addressed in flyteadmin v0.6.20
